activiti-spring-resource-loader/pom.xml

```diff
@@ -0,0 +1,74 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright 2019 Alfresco, Inc. and/or its affiliates.
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~       http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <groupId>org.activiti</groupId>
+        <artifactId>activiti-core-dependencies</artifactId>
+        <version>7.1.0-SNAPSHOT</version>
+        <relativePath>../activiti-core-dependencies</relativePath>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+    <artifactId>activiti-spring-resource-loader</artifactId>
+    <name>Activiti :: Spring Resource Loader</name>
+    <dependencies>
+        <dependency>
+            <groupId>org.activiti.core.common</groupId>
+            <artifactId>activiti-spring-resource-finder</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.activiti</groupId>
+            <artifactId>activiti-engine</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.activiti</groupId>
+            <artifactId>activiti-spring</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.springframework</groupId>
+            <artifactId>spring-beans</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.springframework</groupId>
+            <artifactId>spring-context</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-autoconfigure</artifactId>
```

Comment:
Since the latest changes to remove component scans we realized that it's better to use `spring-boot-autoconfigure` as an optional dependency (see https://github.com/Activiti/Activiti/blob/develop/activiti-spring-process-extensions/pom.xml#L52-L57), except for the starter itself.